Profiles - ZIB 2017/zib-AdministrationSchedule.xml

```diff
@@ -175,8 +175,7 @@
     <element id="Timing.repeat.timeOfDay">
       <path value="Timing.repeat.timeOfDay" />
       <short value="AdministrationTime" />
-      <definition value="The time of administration is a specific time of day (on the clock). This time usually isn’t (intended to be) exact. There can be multiple administering times in one day. The ideal time of administration can also be entered as a time of day (morning, afternoon, evening, night-time). The administration time is then to be left empty, and the time of day can be entered in the TimeOfDay concept." />
-      <comment value="When time of day is specified, it is inferred that the action happens every day (as filtered by dayofWeek) on the specified times. If there's a timeOfDay, there cannot be a when, or vice versa." />
+      <comment value="If there's a timeOfDay, there cannot be a when, or vice versa." />

```

Comment:
Per the style guidelines, this should be:
> If there's a `.timeOfDay`, there cannot be a `.when`, or vice versa.

I would suggest a different wording though:
> If `.timeOfDay` is populated, `.when` SHOULD be absent and vice versa.

In addition, shouldn't a similar comment be also added to `.when`?

Reply:
This is the last part copied from the default FHIR comment. I tried to do a pre-adopt of R5 comment, but we noticed it still contained an error. Therefore Arianne and I decided to only leave the part of the comment that seems correct.

This comment is also on the `.when` in the FHIR spec.

Reply:
Oeh interessant, dus onze NL styleguide is anders dan wat usance is bij FHIR internationaal? Ik vind het idd logisch zo min mogelijk af te wijken van wat FHIR internationaal hier schrijft.

Reply:
Onze styleguide is beterder natuurlijk. Maar goed, laten we dan idd. maar bij de internationale definitie blijven.